scripts/Common.ps1

```diff
@@ -385,6 +385,202 @@ function Create-StorageContainer
     New-AzureStorageContainer -Name $StorageContainerName -Context $storageContext
 }
 
+## Function: Get-AzureStorageContainerStatus
```

Comment:
Switch the function headers to new format

Reply:
Done